project/src/index.tsx

```diff
@@ -2,12 +2,18 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './components/app/app';
 
+const DataOffers = {
+  offersCount: 3,
+} as const;
+
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement,
 );
 
 root.render(
   <React.StrictMode>
-    <App />
+    <App
+      offersCount = {DataOffers.offersCount}
```

Comment:
Форматируем только так
```suggestion
      offersCount={DataOffers.offersCount}
```